src/lib/liblegacy.js

```diff
@@ -116,6 +116,11 @@ legacyFuncs = {
   // Legacy names for runtime `out`/`err` symbols.
   $print: 'out',
   $printErr: 'err',
+
+  // Converts a JS string to an integer base-10, with signaling error
+  // handling (throws a JS exception on error). E.g. jstoi_s("123abc")
+  // throws an exception.
```

Comment:
```suggestion
  // Converts a JS string to an integer base-10. Despite _s, which
  // suggests signaling error handling, this returns NaN on error.
  // (This was a mistake in the original implementation, and kept
  // to avoid breakage.)
```
Might as well fix the docs too?

Reply:
Oops I tried to commit your suggestion (normally I prefer to make the changes locally so I'm not used that UI.. I must have done something wrong).

Reply:
Hmm, maybe it commited ok but your next force-push overwrote it... force-push doesn't warn on that iirc.